components/custom-timetables-list.vue

```diff
@@ -17,32 +17,32 @@
       </v-list-tile-content>
     </v-list-tile>
 
+    <v-list-tile @click="customTimetableDialogOpen = true">
```

Comment:
Würde es besser finden wenn man hier nur die Komponente selbst einbinden würde, also nur `<custom-timetable-dialog v-model="customTimetableDialogOpen"/>` und den Rest in die Komponente selbst einbaut, so haben wir das auch bei Impressum und About gemacht

Reply:
Ja verstehe. Wo wollen wir die Komponente denn wiederverwenden?